packages/volto-separator-block/src/theme/main.less

```diff
@@ -36,6 +40,12 @@
 .block-editor-separator {
   clear: both;
 
+  &.has--shortLine--true .block.separator {
```

Comment:
Where does this come from? 
I'd not do the shortline as in a custom classname, we have the Stylewrapper for it! Even if we only want to inject a classname (not all in the world have to be CSS properties). What do you think?